--- a/config.def.h
+++ b/config.def.h
@@ -63,5 +63,13 @@
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+
+	//{ cpu_perc, "[CPU %s%%]", NULL    },
+	//{ ram_perc, "[RAM %s%%]", NULL    },
+	//{ wifi_perc, "[WIFI %s%%]", "wlp3s0"    },
+	//{ wifi_essid, "[NET %s]", "wlp3s0"    },
+	//{ battery_perc, "[BAT %2s%%]", "BAT0"    },
+	//{ battery_state, "[CHR %s]", "BAT0"    },
+	//{ disk_perc, "[HD %s%%]", "/"    },
+	{ datetime, " %s ",           "%F %T" },
 };
